lib/cell.rb

```diff
@@ -17,4 +18,12 @@ def place_ship(ship_object_parameter)
   def fired_upon?
     false
   end
+
+  def fire_upon
+    if @empty == true
+      "Missed!"
```

Comment:
(As we discussed in person), because we don't know yet what other objects are going to be using Cell's .fire_upon method, we're also not sure what they will be doing with its return value; passing it a string might break something later. My suggestion would be to remove this line (line 24), at least until we know more.

In that case, since we only need something to happen when the cell is _not_ empty, this could be re-written as:
`if !empty?
    @ship.hit
 end`
(Since there's only one instruction we want to conditionally run, we only need one conditional statement; in other words, we can take out the `else` statement.)